src/components/TaskDetails/EditDescription.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, Container, Form } from 'react-bootstrap';
+
+export default class EditDescription extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      editDescription: props.editDescription,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
FYI: https://stackoverflow.com/questions/51980977/why-is-componentwillreceiveprops-deprecated